org.gnome.Epiphany.json

```diff
@@ -25,6 +25,18 @@
                     "commit": "5b7ffe85cded2fa3ce682510706601c1289da004"
                 }
             ],
+    "modules": [
+        {
+            "name": "gstreamer-vaapi",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/GStreamer/gstreamer-vaapi",
+                    "commit": "c3ddb29cb2860374f9efbed495af7b0eead08312"
```

Comment:
Please use a tag instead, its easier to know what version is being used like that.